packages/core/src/tracing/sdk/sdk.js

```diff
@@ -347,11 +328,7 @@ module.exports = function (isCallbackApi) {
     return isCallbackApi ? callback(span) : Promise.resolve(span);
   }
 
-  /**
-   * @param {import('../cls')} _cls
-   */
-  function init(_cls) {
-    cls = _cls;
+  function init() {
```

Comment:
`issue(blocking):` Needs to be reverted, we must not require `cls` unconditionally, see above.

Reply:
Will use https://github.com/instana/nodejs/pull/565#discussion_r912768179